econet_heatpump_water_heater.yaml

```diff
@@ -1,12 +1,17 @@
 substitutions:
   name: "econet-hpwh"
+  friendly_name: "Water Heater"
   device_description: "Rheem Heat Pump Water Heater"
   tx_pin: GPIO19
   rx_pin: GPIO22
 
 esphome:
   name: ${name}
+  friendly_name: ${friendly_name}
   comment: ${device_description}
+  project:
```

Comment:
Remove

Reply:
Is `project` undesired for some reason? It still shows in the spec at https://esphome.io/components/esphome#esphome-creators-project

Reply:
It isn't undesired. The project name you have is for the other repo. And this repo isn't currently using releases and it has no versions. Just drop it for now and we can add it back later with the correct name and version.